domains.txt

```diff
@@ -175,12 +199,17 @@ digitalgurus.co.uk
 digitalpursuit.co.uk
 digitechis.com
 digitechresourcing.co.uk
+digvis.co.uk
```

Comment:
Digital Vision Search are a team of highly experienced Digital recruiters. We work with companies to find the people that are required to build awesome produ